### PR TITLE
Remove 'feature_names' parameter from model.print_feature_importances

### LINKED ISSUE #834: Remove `feature_names` parameter from model.print_feature_importances
The `print_feature_importances` function can simply access the feature names of the model directly (self.get_human_readable_feature_names()), so the `feature_names` argument is not needed.

### PR DESCRIPTION
**Pull request by zephyrmoth**
Fixes #834

### CODE REVIEW

**Comment by chidauri** [comment]
@zephyrmoth can you post the results before/after? It'll be useful.

File: scripts/bug_classifier.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def classify_bugs(model_name, classifier):
             feature_names = model.get_human_readable_feature_names()
```
Comment (marco-c) [inline scripts/bug_classifier.py]: You're not using this anymore, so you should remove it

File: bugbug/model.py [hunk 2]
```diff
@@ -239,7 +239,7 @@ def print_feature_importances(
                 predicted_class_index = class_probabilities.argmax(axis=-1)[0]
             else:
                 predicted_class_index = 0
-
+            feature_names = self.get_human_readable_feature_names()
```
Comment (marco-c) [inline bugbug/model.py]: You are defining this in the `if` branch, which means it won't be available in the `else` where it is used...

**Comment by marco-c** [comment]
Closing PR as it's stale.

**Pull request closed without merging**